### PR TITLE
Fix: Updated j2 templates to fix jinja single-statement-per-line error

### PR DESCRIPTION
**Pull request by ghost**
## Change Summary

Remove general disable of j2lint rule "single-statement-per-line"
Fix templates so they conform.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vmtracer-sessions.md [hunk 1]
```diff
@@ -55,8 +55,10 @@ interface Management1
 
 | Session | URL | Username | Autovlan | Source Interface |
 | ------- | --- | -------- | -------- | ---------------- |
-| session_1 |  https://192.168.0.10 | user1 | disabled |  Management1 |
-| session_2 |  https://192.168.0.10 | user1 | enabled |  - |
+|
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vmtracer-sessions.md]: something is wrong in this output. If you check before and after as rendered markdwon.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2 [hunk 2]
```diff
@@ -8,7 +8,8 @@
 | ------------------| ----- | --- | ---------- |
 {%     for aaa_server_group in aaa_server_groups %}
 {%         for server in aaa_server_group.servers %}
-| {{ aaa_server_group['name'] }} | {{ aaa_server_group['type'] }} | {% if server['vrf'] is defined and server['vrf'] is not none %} {{ server['vrf'] }} {% else %} default {% endif %}| {{ server['server'] }} |
+{%             set vrf = server['vrf'] | arista.avd.default('default') %}
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2]: We should use jinja style when the key is a static string value, so `set vrf = server.vrf ...` same below

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2]: same for other templates

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vmtracer-sessions.md (hunk 1 above)
Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/vmtracer-sessions.md]: Corrected.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2 (hunk 2 above)
Comment (Shivani-chourasiya) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2]: Updated.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Thank you for taking this on.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by KonikaChaurasiya-GSLab — COMMENTED** [review]
LGTM!

**Comment by ClausHolbechArista** [comment]
Please update github workflow so we no longer ignore this rule:
Current: `j2lint $GITHUB_WORKSPACE --ignore S7`

**Comment by Shivani-chourasiya** [comment]
> Please update github workflow so we no longer ignore this rule: Current: `j2lint $GITHUB_WORKSPACE --ignore S7`

Updated!

**Review by KonikaChaurasiya-GSLab — APPROVED** [review]

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**